src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+public class Ball {
+    private int number;
+    private Color color;
+
+    public static String toString(int number, Color color) {
+        return String.valueOf(number) + " " + color.name();
+    }

```

Comment:
You need to add consrtructor with parameters or setters to have ability to create Ball with color and number